client/cypress/support/company-list.po.ts

```diff
@@ -0,0 +1,63 @@
+export class CompanyListPage {
+  private readonly baseUrl = '/companies';
+  private readonly pageTitle = '.company-list-title';
+  private readonly companyCardSelector = '.company-cards-container .company-card';
+  // private readonly userListItemsSelector = '.user-nav-list .user-list-item';
+  // private readonly profileButtonSelector = '[data-test=viewProfileButton]';
+  // private readonly radioButtonSelector = `[data-test=viewTypeRadio] mat-radio-button`;
+  // private readonly userRoleDropdownSelector = '[data-test=userRoleSelect]';
+  // private readonly dropdownOptionSelector = `mat-option`;
+  // private readonly addUserButtonSelector = '[data-test=addUserButton]';
+
+  navigateTo() {
+    return cy.visit(this.baseUrl);
+  }
+
+  /**
+   * Gets the title of the app when visiting the `/users` page.
+   *
+   * @returns the value of the element with the ID `.user-list-title`
+   */
+  getUserTitle() {
+    return cy.get(this.pageTitle);
+  }
+
+  /**
+   * Get all the `company-card` DOM elements.
+   *
+   * @returns an iterable (`Cypress.Chainable`) containing all
+   *   the `company-card` DOM elements.
+   */
+   getCompanyCards() {
+    return cy.get(this.companyCardSelector);
+  }
+
+  /**
+   * Get a specific `company-card` DOM element by its name
+   * (which is in the `_id`) field.
+   */
+  getCompanyCardByName(companyName: string) {
+    return cy
+      // Get all the company cards
+      .get(this.companyCardSelector)
+      // Look for a (sub)element with the company name
+      .contains(companyName)
+      // Return the containing company card element
+      .parents('.company-card');
```

Comment:
I'm not sure I'm thrilled about this "go down to the company name" and then "go up one step to the `.company-card` approach, but I'm not sure of a better way.